stores/message_history_store.jsx

```diff
@@ -44,12 +44,12 @@ class MessageHistoryStoreClass {
             return null;
         }
 
-        if (keyCode === Constants.KeyCodes.UP) {
+        if (keyCode === Constants.KeyCodes.UP[1]) {
```

Comment:
We would need to change the parameters of this function in order to remove keyCode from it. Couldn't find where the parameters are passed though.

Reply:
The message_history_store is a wrapper over the mattermost-redux message history, while all components are migrated to use directly mattermost-redux message_history. I think, now is ready to remove, I have to confirm it.